docker-build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+GOOS=linux GOARCH=amd64 gb build
+
+mkdir -p docker/bin
+cp bin/*linux-amd64 docker/bin/
+
+cd docker
+
+for cli in {client,federation}-api-proxy dendrite-{{client,federation,media,sync}-api,room}-server; do
+    dockerfile=Dockerfile.$cli
+    cat <<EOF > $dockerfile
+FROM scratch
+COPY bin/$cli-linux-amd64 $cli
```

Comment:
Do you have something in your environment that causes non-static go binaries to be built? I cross-compiled on macOS using golang 1.8.3 and got completely static binaries I believe.